src/cmps/button/Button.tsx

```diff
@@ -0,0 +1,26 @@
+//Button component
+
+import React, { MouseEvent } from "react";
+import './Button.css';
+
+// without state use react stateless function
+type props = {
+    isDisabled?: boolean;
+    value: string;
+    onClick(e: MouseEvent<HTMLElement>): void;
+}
+
+const Button: React.FC<props> = ({ isDisabled, value, onClick: handleClick }) => {
+    var styleDisable: string = '';
+    if (isDisabled) { styleDisable = 'default' } else { styleDisable = 'disabled' };
+    const classes: string = `button ${value} ${styleDisable}`;
+    console.log(value);
+    console.log(styleDisable);
+    return (
+        <div className="buttons">
+            <button onClick={handleClick} className={classes}>{value}</button>
```

Comment:
I'm pretty sure that I've seen a more updated version of this file (we refactored together the `handleClick`).
can you pls make sure you pushed all commits?

Reply:
you right... It has been modified already